assets/i18n/qqq.json

```diff
@@ -480,6 +480,7 @@
 	"I18N_SIDEBAR_LIBRARY_LINK": "Text displayed in the side navigation bar. - When the user clicks the link, they are redirected to the Library page.\n{{Identical|Library}}",
 	"I18N_SIDEBAR_OPPIA_FOUNDATION": "Text displayed in the side navigation bar. - When the user clicks the link, they are redirected to the Oppia Foundation site.\n{{Identical|Oppia Foundation}}",
 	"I18N_SIDEBAR_TEACH_WITH_OPPIA": "Text displayed in the side navigation bar. - When the user clicks the link, they are redirected to the Teach page.",
+	"I18N_SIDEBAR_WELCOME_GUIDE_PAGE": "Welcome Guide",
```

Comment:
This should be the explanation of where/when this text will be used.

Reply:
Fixed!